index.html

```diff
@@ -68,7 +68,340 @@ <h3>
     </div>
     </div>
     
-    </section>
+</section>
+
+<section class="recent-works" id="portfolio-section">
+   
+
+  
+  <header class="title-header">
+
+     
+      <h2 class="title">My Recent Works</h2>
+  
+<div><hr class="title-border"></div>
+    </header>
+     <div class="d-flex">
+    <div class="work">
+        <section >
+          <img src="images/bestwork.png" class="desk-img" alt="Best work">
+
+        </section>
+      </div>
+        <div class="mg">
+
+      
+        <section class="stories">
+          <h5>Multi-Post Stories</h5> 
+        </section>
+        <section class="story-details">
+            <p>
+              A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.
+            </p>
+        </section>
+        <ul class="langs">
+            <li>
+                CSS
+              </li>
+            <li>
+                Html
+              </li>
+            <li>Bootstrap</li>
+            <li>Ruby</li>
+        </ul>
+        <section class="disp-btn">
+            <button class="pst-btn" id="showProject">see project</button>
+        </section>
+    </div>
+       </div>
+
+
+<section class="myProjects">
+  <div class="project">
+      <div class="projectHeader">
+          <h2>Profesional Art Printing Data</h2>
+      </div>
+
+      <div class="projectBody">
+          <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+          <ul>
+              <li class="projectHTML">
+                  <ul>
+                      <li>html</li>
+                  </ul>
+              </li>
+              <li class="projectBootstrap">
+                  <ul>
+                      <li>bootstrap</li>
+                  </ul>
+              </li>
+              <li class="projectRuby">
+                  <ul>
+                      <li>ruby</li>
+                  </ul>
+              </li>
+          </ul>
+      </div>
+      <div class="projectButton">
+          <button type="button">See Project</button>
+      </div>
+  </div>
+
+  <div class="project">
+      <div class="projectHeader">
+          <h2>Profesional Art Printing Data</h2>
+      </div>
+
+      <div class="projectBody">
+          <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+          <ul>
+              <li class="projectHTML">
+                  <ul>
+                      <li>html</li>
+                  </ul>
+              </li>
+              <li class="projectBootstrap">
+                  <ul>
+                      <li>bootstrap</li>
+                  </ul>
+              </li>
+              <li class="projectRuby">
+                  <ul>
+                      <li>ruby</li>
+                  </ul>
+              </li>
+          </ul>
```

Comment:
- [ ] kindly align the titles of the technologies in the center to match Figma design  🙏 

![image](https://user-images.githubusercontent.com/89282221/192111100-7360bd34-aecb-436c-b07d-8c1188b324ef.png)